gnomad/gnomadv4_extract_test.py

```diff
@@ -0,0 +1,50 @@
+# LT 23/08/2022
```

Comment:
git already keeps author and time information on a line-by-line basis (+ history).

Reply:
an old habit, I rather stick to it if it doesn't hurt